resteasy/src/main/java/com/cerner/beadledom/resteasy/ResteasyContextListener.java

```diff
@@ -76,4 +70,21 @@ protected void withInjector(Injector injector) {
    * Returns the list of Guice modules to be used with Resteasy.
    */
   protected abstract List<? extends Module> getModules(ServletContext context);
+
+  /**
+   * Processes module bindings in the Guice injector.
+   */
+  @VisibleForTesting
+  public void processInjector(ServletContext context, Injector injector) {
```

Comment:
Removed the while loop: https://github.com/cerner/beadledom/pull/13/commits/b666e0add8a35878245812f04789793a83a02fc6